internal/service/networkpeering/resource_network_peering.go

```diff
@@ -412,11 +414,7 @@ func resourceUpdate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 
 	// Updating any of the attributes for Azure Network Peering forces a recreation of the network peering.
 	// Need to check if GCP and AWS have the same behavior
```

Comment:
about this comment that is previous to this PR, should we check it and possibly include in this PR any new information?

Reply:
Removed the comment since it's no longer relevant